### PR TITLE
Feature/redis plugins config

### PR DESCRIPTION
**Pull request by ArjenDavid-sjtu**
### <Feature description>
- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [x] Update the documentation to include this new feature.
- [x] Tests(including UT, IT, E2E) are added to verify the new feature.
- [ ] If it's UI related, attach the screenshots below.

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes [skywalking-8414](https://github.com/apache/skywalking/issues/8414).
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

**Comment by ArjenDavid-sjtu** [comment]
I am not pretty sure TRACE_REDIS_PARAMETERS should be true or false by default. What's your opinion?

File: apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedissonPluginConfig.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import org.apache.skywalking.apm.agent.core.boot.PluginConfig;
+
+public class RedissonPluginConfig {
+    public static class Plugin {
+        @PluginConfig(root = RedissonPluginConfig.class)
+        public static class Redisson {
+            /**
+             * If set to true, the parameters of the Redis command would be collected.
+             */
+            public static boolean TRACE_REDIS_PARAMETERS = true;
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedissonPluginConfig.java:30]: This should be disabled in default.

File: test/plugin/scenarios/jedis-scenario/bin/startup.sh [hunk 2]
```diff
@@ -18,5 +18,5 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -jar ${agent_opts} -Dredis.host=${REDIS_HOST} -Dredis.port=${REDIS_PORT} \
+java -jar ${agent_opts} -Dredis.host=${REDIS_HOST} -Dredis.port=${REDIS_PORT} -Dskywalking.plugin.jedis.trace_redis_parameters=true\
```
Comment (wu-sheng) [inline test/plugin/scenarios/jedis-scenario/bin/startup.sh:21]: As you activate this, please update the expected segment to verify this new tag.

File: test/plugin/scenarios/redisson-scenario/bin/startup.sh [hunk 3]
```diff
@@ -18,4 +18,4 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -Dredis.servers=${REDIS_SERVERS} -jar ${agent_opts} ${home}/../libs/redisson-scenario.jar &
\ No newline at end of file
+java -Dredis.servers=${REDIS_SERVERS} -Dskywalking.plugin.redisson.trace_redis_parameters=true -jar ${agent_opts} ${home}/../libs/redisson-scenario.jar &
```
Comment (wu-sheng) [inline test/plugin/scenarios/redisson-scenario/bin/startup.sh:21]: As you activate this, please update the expected segment to verify this new tag.

File: CHANGES.md [hunk 4]
```diff
@@ -21,6 +21,7 @@ Release Notes.
 * Compatible with the versions after dubbo-2.7.14
 * Follow protocol grammar fix `GCPhrase -> GCPhase`.
 * Support ZGC GC time and count metric collect. (Require 9.0.0 OAP)
+* Add configuration of collecting redis parameters for jedis-2.x and redisson-3.x plugin.
```
Comment (wu-sheng) [inline CHANGES.md:24]:
```suggestion
* Support collecting redis parameters for jedis-2.x and redisson-3.x plugin.
```

You are adding a feature, not just a config.

File: test/plugin/scenarios/jedis-scenario/bin/startup.sh (hunk 2 above)
Comment (ArjenDavid-sjtu) [reply test/plugin/scenarios/jedis-scenario/bin/startup.sh:21]: Both plugins support collecting redis parameters previously, what I do was to add a switch to disable the collection. So the tag don't need to change.

File: test/plugin/scenarios/redisson-scenario/bin/startup.sh (hunk 3 above)
Comment (ArjenDavid-sjtu) [reply test/plugin/scenarios/redisson-scenario/bin/startup.sh:21]: The same as above.

**Comment by wu-sheng** [comment]
But the Jedis test fails.

**Comment by wu-sheng** [comment]
Please check https://github.com/apache/skywalking-java/runs/4802555245?check_suite_focus=true

It seems the test application can't bootstrap.

File: test/plugin/scenarios/jedis-scenario/bin/startup.sh [hunk 5]
```diff
@@ -18,5 +18,5 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -jar ${agent_opts} -Dredis.host=${REDIS_HOST} -Dredis.port=${REDIS_PORT} -Dskywalking.plugin.jedis.trace_redis_parameters=true\
+java -Dskywalking.plugin.jedis.trace_redis_parameters=true -jar ${agent_opts} -Dredis.host=${REDIS_HOST} -Dredis.port=${REDIS_PORT}\
```
Comment (lujiajing1126) [inline test/plugin/scenarios/jedis-scenario/bin/startup.sh:21]: The problem here is not related to where you place the system property but _the space at the end of the line._

This is the reason you failed to run it locally and the CI reported.

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

**Pull request merged**